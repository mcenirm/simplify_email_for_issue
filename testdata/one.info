Example MIME message from https://msdn.microsoft.com/en-us/library/ms526560(v=exchg.10).aspx
