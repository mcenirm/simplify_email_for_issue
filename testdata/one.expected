From: John Doe <example@example.com>

This is a multipart message in MIME format.

this is the body text

